### DISCUSSION TITLE
compile GUI application without DOS console?

### DISCUSSION THREAD

**Discussion by retsyo** [pascalabcnet/pascalabcnet#2591] [Q&A] [answered]
Is there a way to create GUI application without the black DOS console window?

Thanks

**Comment by SunSerega** [comment, marked as answer]
Use `{$apptype windows}`. It is applied by default in standard graphics units like `GraphWPF`.

Note: Launching program with binding to IDE (aka. just F9) recompiles .exe to allocate console anyway, so that "Input window" of IDE would work.
